src/chexus/validators.py

```diff
@@ -211,6 +211,37 @@ def validate(self, node: Dataset | Group) -> Violation | None:
             return Violation(node.name)
 
 
+class transformation_offset_units_invalid(Validator):
+    def __init__(self) -> None:
+        super().__init__(
+            "transformation_offset_units_invalid",
+            "Transformation offset_units attr. should have length unit if "
+            "transformation_type attr. is 'translation' and "
+            "rotation unit if transformation type is 'rotation'",
```

Comment:
Hmm, I think there is a misunderstanding here. The offset should always have a length unit. The linked issue was mainly about the unit of the value dataset.

Reply:
Aha yes, I misunderstood.

I'll check the units of the value dataset instead.

Reply:
We can also check the `offset_units` attr, but it should always be a length.